config/routes.rb

```diff
@@ -15,6 +15,7 @@
       get :event_descriptions
       post :validate
       post :complete
+      delete :undefined, action: :destroy_undefined
```

Comment:
Right, it's purely cosmetic now the route is`DELETE /agents/undefined`, having destroy in it felt recurring, but I stole the idea from the `delete :memory` route we already have :)